frontend/packages/console-shared/src/hooks/useUserSettingsCompatibility.ts

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { useUserSettings } from './useUserSettings';
+
+export const useUserSettingsCompatibility = (
+  key: string,
+  defaultValue: string = '',
+  storageKey?: string,
```

Comment:
Also when this is @christianvogt recommended order, we should use `useUserSettingsCompatibility` only with a storageKey. So I still recommend `key: string, storageKey: string, defaultValue?: T`

If we replace this later with `useUserSettings` it doesn't matter if you need to drop the 2nd or 3rd argument.

Reply:
I also would recommend to return JSON parsed data if possible?!